share/locale/en.json

```diff
@@ -1,165 +1,165 @@
+    "wait_for_hs_yup": "Ready.",
+    "give_this_url": "Give this address to the recipient of the file:",
+    "give_this_url_stealth": "Give this address and HidServAuth line to the recipient of the file:",
+    "give_this_url_receive": "Give this address to those that send you files:",
+    "give_this_url_receive_stealth": "Give this address and your HidServAuth line to those that send you files:",
     "ctrlc_to_stop":  "Press Ctrl+C to stop the server",
     "not_a_file": "{0:s} is not a valid file.",
     "not_a_readable_file": "{0:s} is not a readable file.",
-    "no_filenames": "You must specify a list of files to share.",
-    "no_available_port": "Could not start the Onion service as there was no available port.",
-    "download_page_loaded": "Download page loaded",
-    "other_page_loaded": "Address loaded",
-    "close_on_timeout": "Stopped because timer expired",
-    "closing_automatically": "Stopped because download finished",
-    "timeout_download_still_running": "Waiting for download to complete",
+    "no_filenames": "Add files to OnionShare.",
+    "no_available_port": "Could not start the onion service without a port to do it on.",
+    "download_page_loaded": "OnionShare page loaded",
+    "other_page_loaded": "OnionShare address loaded",
+    "close_on_timeout": "Timer expiry stopped OnionShare",
+    "closing_automatically": "OnionShare completed, now stopped",
+    "timeout_download_still_running": "Awaiting OnionShare completion",
     "large_filesize": "Warning: Sending large files could take hours",
-    "error_tails_invalid_port": "Invalid value, port must be a regular number",
+    "error_tails_invalid_port": "Wrong value, \"Port\" must be a regular number",
     "error_tails_unknown_root": "Unknown error with Tails root process",
     "systray_menu_exit": "Quit",
-    "systray_download_started_title": "OnionShare Download Started",
-    "systray_download_started_message": "A user started downloading your files",
-    "systray_download_completed_title": "OnionShare Download Finished",
-    "systray_download_completed_message": "The user finished downloading your files",
-    "systray_download_canceled_title": "OnionShare Download Canceled",
-    "systray_download_canceled_message": "The user canceled the download",
+    "systray_download_started_title": "OnionSharing",
+    "systray_download_started_message": "Files underway",
+    "systray_download_completed_title": "OnionSharing complete",
+    "systray_download_completed_message": "The user has your files now",
+    "systray_download_canceled_title": "OnionSharing stopped",
+    "systray_download_canceled_message": "The user stopped reception",
     "help_local_only": "Do not attempt to use Tor: For development only",
-    "help_stay_open": "Keep onion service running after download has finished",
-    "help_shutdown_timeout": "Shut down the onion service after N seconds",
-    "help_stealth": "Create stealth onion service (advanced)",
```

Comment:
"Stealthify" to my chagrin is not a word, and "hidden" is not an option.
This string reads like it should be "stealthy", in effect more-so than it qualifies it as any different.

Reply:
Heh. I kind of like the language, but I wonder if it's misleading. A "stealth onion service" is a specific type of onion service, but also with the new v3 onion services, they are _all_ stealthy now. Really a more accurate phrase might just be "Use client authentication (advanced)"...

Reply:
I agree we should start using the term 'Client Auth' because that is the upstream term, so at least it's consistent.

Reply:
How about "Create OnionShare that additionally requires HidServAuth to fetch"